docs/networking/cilium.md

```diff
@@ -138,6 +138,18 @@ Once the secret has been created, encryption can be enabled by setting `enableEn
       enableEncryption: true
 ```
 
+#### Resources in Cilium
+{{ kops_feature_table(kops_added_default='1.21', k8s_min='1.20') }}
+
+As of kOps 1.20, it is possible to choose your own values for Cilium Agents + Operator. Example:
+```yaml
+  networking:
+    cilium:
+      cpuRequest: "25m"
+      memoryRequest: "128Mi"
+      cpuLimit: "100m"
```

Comment:
Setting limits on system pods is a bit risky. I'd rather not allow it. It is better that cilium consumes more memory than expected rather than that it OOMs.

Reply:
removed limits ^^